cross/par2cmdline/patches/88f6281/c++11-to-c++0x.patch

```diff
@@ -0,0 +1,24 @@
+diff -uprN ../par2cmdline-0.8.1.orig/Makefile.am ./Makefile.am
+--- ../par2cmdline-0.8.1.orig/Makefile.am	2020-02-09 14:29:09.000000000 +0000
++++ ./Makefile.am	2022-08-29 10:53:46.532922839 +0000
+@@ -57,7 +57,7 @@ par2_SOURCES = src/par2cmdline.cpp \
+ par2_LDADD = libpar2.a
+ 
+ LDADD = -lstdc++
+-AM_CXXFLAGS = -Wall -std=c++11 $(OPENMP_CXXFLAGS)
++AM_CXXFLAGS = -Wall -std=c++0x $(OPENMP_CXXFLAGS)
```

Comment:
Python 3.11 requires C++17, so for a moment I thought this would exclude the 88f6281 arch when Python 3.11 is released. 
But seems it should still work, if the right flags are used.

Reply:
C++17 requirement will drop all DSM <7 builds.

Reply:
Typo, meant C++11!

Reply:
even `c++11` will require dropping `gcc<4.9`.

BTW @Safihre I'm pushing this PR as I noticed a bug in `service-setup.sh` where the `incomplete`, `complete` and `watch` directories under `downloads` are not group write permitted which impacts other packages.  I've come-up with a method to alleviate that even on DSM7 such as https://github.com/SynoCommunity/spksrc/pull/5398/files#diff-0be67c7c2427a76d082fddbe7509dbf6111868c62f50d755bb1318131fa7dc1a

Therefore I took the opportunity to create an updated package so any new install do not block other packages.

But other than that this package served me as a perfect example to nail down `deluge` hopefully once and for all :)